packages/atomic/src/components/search/atomic-generated-answer/atomic-generated-answer.tsx

```diff
@@ -54,9 +62,16 @@ export class AtomicGeneratedAnswer implements InitializableComponent {
   @Element() private host!: HTMLElement;
 
   private stopPropagation?: boolean;
+  private storage: SafeStorage = new SafeStorage();
+  private data?: GeneratedAnswerData;
 
   public initialize() {
-    this.generatedAnswer = buildGeneratedAnswer(this.bindings.engine);
+    this.data = this.readStoredData();
+    this.generatedAnswer = buildGeneratedAnswer(this.bindings.engine, {
```

Comment:
If `initialState.isVisible` is initially `false`, will an answer still be generated under the hood, and simply not displayed?

If that is the case, I feel like this is a fairly costly operation to perform for users who don't care for generated answers 😅 

Ideally, I'd expect that if the switch is initially off, then no generated answer would be requested, and if I turn it on, a generated answer would be requested at that point.

Reply:
It's funny, I asked the exact same question when implementing the feature :smile: .
Long story short, the switch must be seen as a way to expand and collapse the answer more than a way to enable or disable the answer generation. In fact, Search API callers cannot control whether answers are going to be generated. Answers are generated based on the query pipeline configuration.

Reply:
OK, wasteful, but makes sense 😄